src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
-body {
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 100px;
```

Comment:
The `$calendar-gap` variable is calculated as a fraction of `$calendar-size`, which is not a typical approach for defining gaps. Consider using a fixed unit like pixels or ems for gaps to ensure consistency and predictability in the layout.